vnext/Microsoft.ReactNative.IntegrationTests/ReactNativeHostTests.cpp

```diff
@@ -5,15 +5,25 @@
 #include <NativeModules.h>
 #include <winrt/Windows.System.h>
 #include "MockReactPackageProvider.h"
+
 using namespace React;
 
 namespace ReactNativeIntegrationTests {
 
 REACT_MODULE(TestHostModule)
 struct TestHostModule {
   REACT_INIT(Initialize)
-  void Initialize(ReactContext const & /*reactContext*/) noexcept {
+  void Initialize(ReactContext const &reactContext) noexcept {
     TestHostModule::Instance.set_value(*this);
+
+    bool jsiExecuted{false};
+    reactContext.ExecuteJsi([&](facebook::jsi::Runtime &rt) {
+      jsiExecuted = true;
+      auto eval = rt.global().getPropertyAsFunction(rt, "eval");
```

Comment:
Should probably use something other than eval for this, since eval wont work in hermes.

Reply:
Let me see if I can replace it with loading function from a string instead.

Reply:
Actually, I have copied this code from JSI unit tests. I would expect Hermes to pass them. I will keep this test code "as-is" for now.